src/main/java/com/github/manolo8/darkbot/config/Config.java

```diff
@@ -182,9 +183,17 @@
         public @Option boolean DEV_STUFF = false;
         public @Option @Editor(JListField.class) @Options(ApiSupplier.class) int API = 2;
         public @Option boolean FULLY_HIDE_API = true;
+        public @Option boolean SAVE_GUI_POS = false;
         public @Option boolean ENFORCE_HW_ACCEL = true;
         public @Option boolean DISABLE_MASTER_PASSWORD = false;
 
+        public Window MAIN_GUI_WINDOW = new Window();
+        public static class Window {
```

Comment:
```suggestion
        public WindowPosition MAIN_GUI_WINDOW = new WindowPosition();
        public static class WindowPosition {
```
`Window` is the name of an AWT class/concept, naming a class of your own (which has no real relation to a window) `Window` is not a good idea, you need to make sure whoever comes next sees clearly that this is a WindowPosition or WindowLocation or somthing along the lines, not a Window